### PR TITLE
Fixes grills. they went too far this time.

### PR DESCRIPTION
**Pull request by ghost**
:cl: Qustinnus
 fix: Whoever the damn bastard is who tried to take our grills. know this: we will not bend over while you take the only thing that still makes us happy. We just want to drink energy drinks and grill some hamburgers in peace. This is your final warning, try doing this again and we will assemble a lawn mower squad outside of your house at the early hours of 6AM to make sure you cannot enjoy your sunday morning in peace.
/:cl:

(newfood compatibility is now included by checking IS_EDIBLE() and removing the shitty foodtype that was there before)

### CODE REVIEW

**Comment by MrMelbert** [comment]
fixes #54349 and empowers me as a radical centrist

File: code/modules/food_and_drinks/kitchen_machinery/grill.dm [hunk 1]
```diff
@@ -43,31 +43,30 @@
...
-				I.reagents.remove_reagent(/datum/reagent/consumable/monkey_energy, I.reagents.get_reagent_amount(/datum/reagent/consumable/monkey_energy))
-				update_icon()
-				return
+	if(istype(I, /obj/item/reagent_containers/food/drinks))
+		if(I.reagents.has_reagent(/datum/reagent/consumable/monkey_energy))
+			grill_fuel += (20 * (I.reagents.get_reagent_amount(/datum/reagent/consumable/monkey_energy)))
+			to_chat(user, "<span class='notice'>You pour the Monkey Energy in [src].</span>")
+			I.reagents.remove_reagent(/datum/reagent/consumable/monkey_energy, I.reagents.get_reagent_amount(/datum/reagent/consumable/monkey_energy))
+			update_icon()
+			return
```
Comment (Fikou) [inline code/modules/food_and_drinks/kitchen_machinery/grill.dm:52]: this whole thing is my dumb code with requiring specifically food/drinks and not allowing beakers
maybe just do if is edible and then else if reagent containers with this piece of code

Comment (Fikou) [reply code/modules/food_and_drinks/kitchen_machinery/grill.dm:52]: it was my dumb code so it would be oversight fix not balance change 😏

Comment (Qustinnus) [reply code/modules/food_and_drinks/kitchen_machinery/grill.dm:52]: understandable

**Pull request merged**